src/bridges/erc4626/ERC4626Bridge.sol

```diff
@@ -0,0 +1,100 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 Aztec.
+pragma solidity >=0.8.4;
+
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import {IERC4626} from "@openzeppelin/contracts/interfaces/IERC4626.sol";
+import {IRollupProcessor} from "../../aztec/interfaces/IRollupProcessor.sol";
+import {AztecTypes} from "../../aztec/libraries/AztecTypes.sol";
+import {BridgeBase} from "../base/BridgeBase.sol";
+import {ErrorLib} from "../base/ErrorLib.sol";
+import {IWETH} from "../../interfaces/IWETH.sol";
+
+/**
+ * @title Aztec Connect Bridge for ERC4626 compatible vaults
+ * @author johhonn (on github) and the Aztec team
+ * @notice You can use this contract to issue or redeem shares of any ERC4626 vault
+ */
+contract ERC4626Bridge is BridgeBase {
+    using SafeERC20 for IERC20;
+
+    IWETH public constant WETH = IWETH(0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2);
+
+    /**
+     * @notice Sets the address of RollupProcessor.sol
+     * @param _rollupProcessor Address of RollupProcessor.sol
+     */
+    constructor(address _rollupProcessor) BridgeBase(_rollupProcessor) {}
+
+    receive() external payable {}
+
+    fallback() external payable {}
```

Comment:
Why the explicit fallback? Which cases do you expect to hit that is not covered by the receive?